controllers/webhookcerts/controller.go

```diff
@@ -80,6 +82,7 @@ type ReconcileWebhookCertificates struct {
 	logger    logr.Logger
 	namespace string
 	now       time.Time
+	config    *rest.Config
```

Comment:
Why do we need this here?

Reply:
No longer required, was used to get a discoveryClient for checking package versions